README.md

```diff
@@ -84,9 +84,10 @@ Head over to [releases](https://github.com/lite-xl/lite-xl/releases) and downloa
 ### Windows
 
 Lite XL comes with installers on Windows for typical installations.
-Alternatively, we provide ZIP archives that you can download and extract it anywhere and run directly.
+Alternatively, we provide ZIP archives that you can download and extract anywhere and run directly.
 
-To make the installation portable, simply create a `user` folder where `lite-xl.exe` is located.
+To make Lite XL portable (e.g. running Lite XL from a thumb drive),
+simply create a `user` folder where `lite-xl.exe` is located.
```

Comment:
I wonder if we should explain that every config and plugin will now have to be put here.